internal/receiver/simpleprometheusremotewritereceiver/config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simpleprometheusremotewritereceiver
+
+import (
+	"errors"
+	"time"
+
+	"github.com/jaegertracing/jaeger/pkg/multierror"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config/confignet"
+)
+
+var _ component.Config = (*Config)(nil)
+
+const (
+	typeString = "simpleprometheusremotewrite"
```

Comment:
```suggestion
	typeString = "prometheusremotewrite"
```
Ok if we reuse the existing exporter name? The simple prometheus receiver is a wrapper for the prometheus receiver w/ reduced config requirements. I'd suggest not starting w/ "simple" out of the gate since there's no relation.

Reply:
I'm +1 with that suggestion